<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibrariesForm</class>
 <widget class="QWidget" name="LibrariesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Список библиотек</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableView" name="librariesTable">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="installButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Установить выбранную версию</string>
          </property>
          <property name="statusTip">
           <string>Заменяет текущую рабочую версию библиотеку на выбранную</string>
          </property>
          <property name="text">
           <string>Установить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="downloadButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Скачать выбранную версию</string>
          </property>
          <property name="statusTip">
           <string>Скачивает выбранную версию в папку для загрузок (если библиотека находится в файловой системе, то она будет просто скопирована)</string>
          </property>
          <property name="text">
           <string>Скачать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Удалить выбранную версию</string>
          </property>
          <property name="statusTip">
           <string>Удаляет выбранную версию библиотеки. Если выбрана рабочая версия, то необходимо установить новую версию библиотеки</string>
          </property>
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="updateButton">
          <property name="toolTip">
           <string>Обновить список</string>
          </property>
          <property name="statusTip">
           <string>Обновляет состояние всех источников и библиотек в них</string>
          </property>
          <property name="text">
           <string>Обновить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
